src/Symfony/Installer/NewCommand.php

```diff
@@ -105,19 +105,14 @@ protected function execute(InputInterface $input, OutputInterface $output)
     protected function checkSymfonyVersionIsInstallable()
     {
         // 'latest' is a special version name that refers to the latest stable version
-        // available at the moment of installing Symfony
-        if ('latest' === $this->version) {
-            return $this;
-        }
-
         // 'lts' is a special version name that refers to the current long term support version
-        if ('lts' === $this->version) {
+        if (in_array($this->version, array('latest', 'lts'))) {
             return $this;
         }
 
         // validate semver syntax
-        if (!preg_match('/^2\.\d(?:\.\d{1,2})?$/', $this->version)) {
-            throw new \RuntimeException('The Symfony version should be 2.N or 2.N.M, where N = 0..9 and M = 0..99');
+        if (!preg_match('/^2\.\d(?:\.\d{1,2})?(-(dev|BETA)\d*)?$/', $this->version)) {
```

Comment:
In 9b3cdc1 we no longer allow numbers after the `dev` version.

Regarding the `BETA` uppercase/lowercase issue, I thought about that but I decided to leave it untouched. If we silently fix `beta` as `BETA`, we should also fix `DEV` to `dev`. Honestly, I don't know if this is right for user experience or it's bad because we are supporting wrong version names.